portfolio/src/main/webapp/index.html

```diff
@@ -19,5 +19,15 @@
         <img src="/images/selfie.jpeg" alt="My Selfie">
         <h1>Jiaxi Chen</h1>
     </div>
+    <div class="content" id="myIntro">
+        </p><strong>Huge welcome to my portfolio!</strong> My name is Jiaxi Chen, 
```

Comment:
curious, why using </p> rather than <p>?

Reply:
My bad. I've fixed that typo. Thanks!